src/main/java/com/segment/analytics/android/integrations/intercom/IntercomIntegration.java

```diff
@@ -271,7 +271,9 @@ private void setUserAttributes(Traits realTraits, @Nullable Map<String, Object>
 
   private Company setCompany(Map<String, Object> payload) {
     Company.Builder company = new Company.Builder();
+    if (!payload.containsKey("id")) return company.build();
```

Comment:
I would add a test case to see what happens when you set an empty company

Reply:
Gotcha - we actually do, test class line 175!